translations/recommendations.tex/pl.po

```diff
@@ -121,7 +123,7 @@ msgstr ""
 "Twój przewidywany minimalny czas gry wyniesie około 3 godzin.\n"
```

Comment:
```suggestion
"Twój przewidywany minimalny czas gry wyniesie około 3 godziny.\n"
```

Reply:
Około trzech godzin
Około trzy godziny

Obie formy są poprawne?

Reply:
W takim razie słownie "trzech", bo cyfrę czyta się jako "trzy", a nie "trzech"

Reply:
Jak tak teraz patrzę, to tak. "trzech graczy" też słownie

Reply:
Dobra, to zmieniam na słownie tu i tu.